app/models/app.constants.tsx

```diff
@@ -1 +1,2 @@
-export const AUTHENTICATED = true;
```

Comment:
mudaria esse path
pra mim nao faz mt sentido `constants` estar dentro de `models`
E se forem constants usadas no back, eu moveria pra um .env por ex, para o session secret

Reply:
we do not need this anymore